docs/reference/operators/otel-metrics.md

```diff
@@ -2,8 +2,38 @@
 title: otel-metrics
 ---
 
-TODO
+The otel-metrics operator handles collecting and exporting metrics using the
+[Prometheus exporter](https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/prometheus/). When setting
+`otel-metrics-listen=true`, it will serve http requests on "0.0.0.0:2224" (configurable using
+`otel-metrics-listen-address`) with the metrics available at "/metrics".
 
-## Parameters
+## Global Parameters
 
-TODO
+### `otel-metrics-listen`
+
+Enables the Prometheus exporter on the address given by `otel-metrics-listen-address` if set to `true`.
+
+Default: `false`
+
+### `otel-metrics-listen-address`
+
+The listen address that should be serving Prometheus requests.
+
+Default: `0.0.0.0:2224`
+
+## Instance Parameters
+
+### `otel-metrics-name`
+
+Overrides the name of a datasource and explicitly sets it as name for the export. This is mandatory if you
```

Comment:
I didn't understand why it's mandatory in such a case, but I haven't tried the operator yet. Let me try it and come back here.

Reply:
Mandatory to make sure _you mean it_. Otherwise gadget runs (same gadget, different parameters for example) would create metrics with the same name, export it and then mess up existing data. This way you _explicitly_ tell it to export the metrics with a certain (stable) name.